app/javascript/components/home/HomePage.jsx

```diff
@@ -1,26 +1,41 @@
 import React from 'react';
-import { Form } from 'react-bootstrap';
+import { Card, Form, Stack } from 'react-bootstrap';
 import ButtonLink from '../shared_components/utilities/ButtonLink';
 import Spinner from '../shared_components/utilities/Spinner';
 import useEnv from '../../hooks/queries/env/useEnv';
+import Logo from '../shared_components/Logo';
 
 export default function HomePage() {
   const { isLoading, data: env } = useEnv();
   if (isLoading) return <Spinner />;
 
-  return (
-    env.OPENID_CONNECT ? (
+  if (env.OPENID_CONNECT) {
+    return (
+    // env.OPENID_CONNECT ? (
       <Form action="/auth/openid_connect" method="POST" data-turbo="false">
         <input type="hidden" name="authenticity_token" value={document.querySelector('meta[name="csrf-token"]').content} />
         <input variant="brand" className="btn mx-2" type="submit" value="Sign In" />
         <input variant="brand-backward" className="btn" type="submit" value="Sign Up" />
       </Form>
+    );
+  }
```

Comment:
This should be in place of the sign up and sign in buttons below

Reply:
Sorry I did not test this one thoroughly